internal/pkg/override/cdk.go

```diff
@@ -172,6 +182,92 @@ func (cdk *CDK) cleanUp(in []byte) ([]byte, error) {
 	return out.Bytes(), nil
 }
 
+type packageManager struct {
+	name     string
+	lockFile string
+}
+
+var packageManagers = []packageManager{ // Alphabetically sorted based on name.
+	{
+		name:     "npm",
+		lockFile: "package-lock.json",
+	},
+	{
+		name:     "yarn",
+		lockFile: "yarn.lock",
+	},
+}
+
+func (cdk *CDK) installedPackageManager() ([]string, []error) {
```

Comment:
I am not completely sure if this works.

can we use `multierror.Append()` instead of returning `[]error`

Reply:
good point, I addressed the feedback using `errors.Join`https://pkg.go.dev/errors#example-Join